cli/util/upload_bin.go

```diff
@@ -32,8 +32,13 @@ import (
 )
 
 const (
-	// Number of objects to search in a batch for finding max block in container.
-	searchBatchSize = 10000
+	// Number of objects to upload in a batch.
+	// Uploader is able to create only one last incomplete batch with less
+	// than uploadBatchSize objects.
```

Comment:
All batches of `searchBatchSize` size except the most recent one are guaranteed to be completed and don't contain gaps.